cfgen/cfgen

```diff
@@ -2004,9 +2018,16 @@ def aux_pool_list(pool_desc: Dict[str, Any], m0conf: Dict[Oid, Any]) -> None:
     # Generate the node combinations considering node failures as
     # allowed by node tolerance values
     temp_pool_nodes = []
-    for nodes in pool['disk_refs']:
-        temp_pool_nodes += [
-            nodes['node']]
+
+    disk_refs = get_pool_diskrefs(pool)
+    if disk_refs is None:
```

Comment:
```suggestion
    # We need to generate disk_refs only if it is not present in pool description
    if not get_pool_diskrefs(pool):
```

Reply:
@knekrasov  Here, disk_refs  is used after checking so this changes is not needed.
I have applied your suggestion at previous comment

Reply:
Ah, correct, I put the suggestion to the wrong line. Apologies :-)